devito/parameters.py

```diff
@@ -45,3 +47,4 @@ def _updated(self, key, value):
 parameters = Parameters()
 # Default log level set to INFO
 parameters["log_level"] = 'INFO'
+parameters["autotuning"] = os.environ.get('DEVITO_ATMODE', 'basic')
```

Comment:
Turned out this was more tangled than I thought.

I eventually opted for a generalization effort, using the global config as expected all over Devito, introducing some nice features, and refactoring the code where needed. Please take a look; it should be much better now. Thanks for pointing it out